README.md

```diff
@@ -4,10 +4,92 @@
 ![unit-tests](https://github.com/goverland-labs/goverland-datasource-snapshot/workflows/unit-tests/badge.svg)
 ![golangci-lint](https://github.com/goverland-labs/goverland-datasource-snapshot/workflows/golangci-lint/badge.svg)
 
+Goverland Datasource Snapshot is a micro service designed to manage and synchronize various data sources for Goverland ecosystem.
+At the moment the focus is on Snapshot API.
+The microservice also leverages technologies such as gRPC and NATS.
```

Comment:
I think these details should be moved to other section (1-2 lines are base description, another lines are details about technoligies which were used)

Reply:
any ideas where?